cpp/include/Ice/OutgoingAsync.h

```diff
@@ -682,7 +682,7 @@ class AsyncCallback : public GenericCallbackBase
     AsyncCallback(const TPtr& instance, Callback cb, Callback sentcb = 0) :
         _callback(instance), _completed(cb), _sent(sentcb)
     {
-        checkCallback(instance, cb != 0);
+        checkCallback(instance != 0, cb != 0);
```

Comment:
We should try to use `nullptr`.
```suggestion
        checkCallback(instance != nullptr, cb != nullptr);
```

Reply:
I tried, it doesn't work.